tests/src/test/java/org/neo4j/graphalgo/impl/metaPathComputationTests/ComputeAllMetaPathsSchemaFullTest.java

```diff
@@ -81,25 +83,51 @@ public static void shutdownGraph() throws Exception {
 
     @Before
     public void setupMetaPaths() throws Exception {
-        algo = new ComputeAllMetaPathsSchemaFull(3, api);
-        HashMap<Integer, String> idTypeMappingNodes = new HashMap<>();
-        HashMap<Integer, String> idTypeMappingEdges = new HashMap<>();
-        idTypeMappingNodes.put(-1, "A");
-        idTypeMappingNodes.put(-2, "B");
-        idTypeMappingNodes.put(-3, "C");
-        idTypeMappingEdges.put(-10, "TYPE1");
-        algo.setIDTypeMappingNodes(idTypeMappingNodes);
-        algo.setIDTypeMappingEdges(idTypeMappingEdges);
+
     }
 
-    @Ignore //TODO could be a problem if we consider the direction of edges
+    @Ignore//TODO could be a problem if we consider the direction of edges//comment still relevant?
```

Comment:
You can write a test like in computeAllMetaPathsSchemaFull on the branch testsForComputeAllMetaPathsSchemaFull.